docs/learning/display/Lighting.md

```diff
@@ -0,0 +1,29 @@
+# Lighting, materials, and environment
+
+The [iTwin.js renderer](./index.md) is designed for efficient visualization of large infrastructure digital twins in a constrained browser environment. Photorealistic rendering is **not** part of that design - many of the techniques required for producing more lifelike images are unsupported or impractically expensive using WebGL. Solutions like [LumenRT](https://www.bentley.com/en/products/brands/lumenrt) and [Unity](https://unity.com) can apply more advanced rendering techniques to iModels. Advanced desktop rendering of iModels can be achieved by using [IModelDb.exportGraphics]($backend) to produce meshes in a format the target application can consume.
```

Comment:
could add unreal to the list here.
Also maybe we should link to our unity sample and unreal scene export app?
https://github.com/iTwin/imodel-unity-example
https://developer.bentley.com/unreal

Reply:
The info about alternative iTwin rendering options didn't really belong here. I moved it to a new section in index.md and reworded it.

Reply:
why remove the lumenRT plug in the new section?

Reply:
Because although I'm 90% certain LumenRT can open and display iModels, I can't seem to find a link that will provide evidence of that.

Reply:
I was also 89% sure it could, but fair... resolving